content/docs/guides/app_onboarding/app_protocol_selection.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Application Protocol Selection"
+description: "Application Protocol Selection"
+type: docs
+---
+
+# Application Protocol Selection
+
+OSM is capable of routing different application protocols such as `HTTP`, `TCP`, and `gRPC` differently. The following guide describes how to configure service ports to specify the application protocol to use for traffic filtering and routing.
```

Comment:
nit:
```suggestion
OSM is capable of routing different application protocols such as `HTTP`, `TCP`, and `gRPC`. The following guide describes how to configure service ports to specify the application protocol to use for traffic filtering and routing.
```

Reply:
The word `differently` is important here to note that different app protocols are not routed the same way, despite the fact that they could. For example, HTTP traffic can be routed as TCP traffic, but it can be routed differently as HTTP traffic instead. Is there a different way to convey this more accurately?

Reply:
Maybe this would work? OSM supports distinct routing configurations for application protocols such as `HTTP`, `TCP`, and `gRPC`.

Reply:
@jaellio sounds good. Feel free to create a PR to update this.